packages/examples/examples/insights/README.md

```diff
@@ -6,9 +6,3 @@ This Snap demonstrates how to use the transaction insights feature.
 
 - Babel is used for transpiling TypeScript to JavaScript, so when building with the CLI,
   `transpilationMode` must be set to `localOnly` (default) or `localAndDeps`.
-- For the global `wallet` type to work, you have to add the following to your `tsconfig.json`:
```

Comment:
Can you open an issue or something to remove this from the templates once we release this PR?